REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+inference_manager
+ingestor_images
+ingestor_works
+matcher
+merger
+palette_inferrer
+relation_embedder
+router
+transformer_calm
+transformer_mets
+transformer_miro
+transformer_sierra
+transformer_tei
+
+Create deployment? [y/N]: y
+
+Deployment Summary
+Requested by: arn:aws:sts::760097843905:assumed-role/platform-superdev/KennyR@Wellcomecloud.onmicrosoft.com
+Date created: 2021-07-08T15:23:55.267305
+Deploy data:  /Users/robertkenny/.local/share/weco-deploy/deployment_results/deploy_2021-07-08_16-23-55.json
+
+image ID               summary of changes
+---------------------  --------------------
+aspect_ratio_inferrer  ECR tag updated,
+batcher                ECR tag updated,
+feature_inferrer       ECR tag updated,
+id_minter              ECR tag updated,
+inference_manager      ECR tag updated,
+ingestor_images        ECR tag updated,
+ingestor_works         ECR tag updated,
+matcher                ECR tag updated,
+merger                 ECR tag updated,
+palette_inferrer       ECR tag updated,
+relation_embedder      ECR tag updated,
+router                 ECR tag updated,
+transformer_calm       ECR tag updated,
+transformer_mets       ECR tag updated,
+transformer_miro       ECR tag updated,
+transformer_sierra     ECR tag updated,
+transformer_tei        ECR tag updated,
+
+Deployed release eeb9482f-0a6a-4211-8de9-27cee7567134 to EXAMPLE-PIPELINE (Environment(id='EXAMPLE-PIPELINE', name='EXAMPLE-PIPELINE'))
+
+Checking deployment of eeb9482f-0a6a-4211-8de9-27cee7567134 to EXAMPLE-PIPELINE
+Allowing 600s for deployment.
+
+Deployment of eeb9482f-0a6a-4211-8de9-27cee7567134 to EXAMPLE-PIPELINE successful
+Deployment took 2s
+```
+
+This operation **only tags ECR images with the new environment** it does not deploy any services. 
+
+This will be done when you terraform a new pipeline.
+
+### Terraform a new pipeline
+
+You will now need to create a new pipeline module in [./pipeline/terraform/main.tf](./pipeline/terraform/main.tf).
+
+Copy and paste an exisiting pipeline making sure to update the fields:
+
+- `pipeline_date`: References secrets required to access ES and to sets internal infrastructure labels.
+- `release_label`: Sets the ECR label to use on the service deployment images, created in the above deployment process. 
+- `is_reindexing`: Sets ES cluster/service scaling limits while reindexing, and connects reindexing topics. Enabling this will incur above normal costs for a pipeline.
+
+See the following example:
+
+```tf
+module "catalogue_pipeline_2021-07-06" {
+  source = "./stack"
+
+  pipeline_date = "2021-07-06"
+  release_label = "2021-07-06"
+
+  is_reindexing = false
+
+  # Boilerplate that shouldn't change between pipelines.
+  # ...
+}
+
+module "catalogue_pipeline_MY-NEW-PIPELINE" {
+  source = "./stack"
+
+  pipeline_date = "MY-NEW-PIPELINE"
+  release_label = "MY-NEW-PIPELINE"
+
+  is_reindexing = true
+
+  # Boilerplate that shouldn't change between pipelines.
+  # ...
+}
+```
+
+### Running the reindex script
```

Comment:
Can we pick a better name for this operation? What does run-in the reindex script do?